Sources/SwiftFusion/Inference/AnyArrayBuffer+Differentiable.swift

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 The SwiftFusion Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Contiguous storage for factor graph values (variable assignments, error vectors) of statically
+/// unknown type.
+
```

Comment:
This file comment looks wrong because this isn't the file that declares the storage types. Maybe just delete it?

```suggestion
```